docs/src/parameter_distributions.md

```diff
@@ -61,13 +81,241 @@ We provide some ConstraintTypes, which apply different transformations internall
 Users can also define their own transformations by directly creating a `ConstraintType` object with their own mappings.
 
 !!! note
-    It is up to the user to ensure their provided transforms are inverses of each other.
+    It is up to the user to ensure any custom mappings `transform_constrained_to_unconstrained` and `transform_unconstrained_to_constrained` are inverses of each other.
 
 
 ## 3. The name
 
 This is simply an identifier for the parameters later on.
 
+## 4. The power of the normal distribution
+
+The combination of different normal distributions with these predefined constraints, provides a suprising breadth of prior distributions.
+
+!!! note
+    We **highly** recommend users start with Normal distribution and predefined constraint: these offer the best computational benefits and clearest interpretation of the methods.
+
+For each for the predefined ConstraintTypes, we present animations of the resulting prior distribution for
+```julia
+distribution = Parameterized(Normal(mean,sd))
+```
+where: 
+1. We fix the mean value to 0, and range over the standard deviation
+2. We fix the standard deviation to 1 and range over the mean value
+
+### Without constraints: `constraint = [no_constraints()]`
+
+```@setup no_constraints
+using Distributions 
+using Plots 
+N = 50 
+x_eval = collect(-5:10/200:5) 
+mean_varying = collect(-3:6/(N+1):3) 
+sd_varying = collect(0.1:2.9/(N+1):3) 
+# no constraint 
+unconstrained_to_constrained(x) = x
+
+mean0norm(n)= pdf(Normal(0,sd_varying[n]),x_eval)
+sd1norm(n)= pdf(Normal(mean_varying[n],1),x_eval) 
+constrained_x_eval = unconstrained_to_constrained.(x_eval) 
+p1 = plot(constrained_x_eval, mean0norm.(1)) 
+vline!([unconstrained_to_constrained(0)]) 
+p2 = plot(constrained_x_eval, sd1norm.(1)) 
+vline!([unconstrained_to_constrained(mean_varying[1])]) 
+
+p = plot(p1,p2, layout=(1,2), size = (900,450), legend=false) 
+ 
+anim_unbounded = @animate for n = 1:size(mean_varying)[1] 
+   #set new y data 
+   p[1][1][:y] = mean0norm(n) 
+   p[1][:title] = "Transformed Normal(0," * string(round(sd_varying[n], digits=3)) * ")" 
+   p[2][1][:y] = sd1norm(n) 
+   p[2][2][:x] = [ unconstrained_to_constrained(mean_varying[n]),
+                   unconstrained_to_constrained(mean_varying[n]),       
+                   unconstrained_to_constrained(mean_varying[n])]
+
+   p[2][:title] = "Transformed Normal(" * string(round(mean_varying[n], digits=3)) * ", 1)"
+end 
+```
+```@example no_constraints
+gif(anim_unbounded, "anim_unbounded.gif", fps = 5) # hide
+```
+The following commands generate the transformed `Normal(0.5,1)` distribution
+
+```@repl
+using EnsembleKalmanProcesses.ParameterDistributionStorage
+using Distributions 
+distribution = Parameterized(Normal(0.5,1)) 
+constraint = [no_constraint()]
+name = "unbounded_parameter"
+prior = ParameterDistribution(distribution,constraint,name)
```

Comment:
I agree. Is the purpose to have a nice copy-paste example? Or do you want the reader to see the output? I think my preference would be to add ``;``

Reply:
@trontrytel  I used an REPL environment because I thought you wished users to see what the REPL would show. I'm happy to put in a non-REPL environment

Reply:
I'm just advocating for the semi-colons because you can verbatim copy the lines into REPL (without removing the `julia>` prefix on each line) to run every statement. If you include the output, then this is not possible.
```julia
julia> using EnsembleKalmanProcesses.ParameterDistributionStorage;

julia> using Distributions;

julia> distribution = Parameterized(Normal(0.5,1));

julia> constraint = [no_constraint()];

julia> name = "unbounded_parameter";

julia> prior = ParameterDistribution(distribution,constraint,name);
```

Reply:
I guess this works fine, although it might be cleaner then just as:

_One can paste the following block into the REPL_
```julia  
using EnsembleKalmanProcesses.ParameterDistributionStorage
using Distributions
distribution = Parameterized(Normal(0.5,1))
constraint = [no_constraint()]
name = "unbounded_parameter"
prior = ParameterDistribution(distribution,constraint,name)
```
Or woudl you still like to see the "Julia" prompt? I guess i thought point of the REPL environment is to show the output from the REPL, and not just the code (e.g as above)

Reply:
I will do this for now, and you let me know how it looks!

Reply:
I stand corrected. You can keep both `Julia>` and the output. If the entire code block is copied and pasted, only the lines with `Julia>` should be executed. Ref the Julia docs: https://docs.julialang.org/en/v1/stdlib/REPL/#The-Julian-mode

> In Julia mode, the REPL supports something called prompt pasting. This activates when pasting text that starts with julia> into the REPL. In that case, only expressions starting with julia> are parsed, others are removed. This makes it possible to paste a chunk of code that has been copied from a REPL session without having to scrub away prompts and outputs.

Reply:
I think you can revert back to the original format, then add text along the lines of

_you can copy the following block and paste it directly to the REPL to inspect the output (note that only blocks prefixed by `julia>` will be executed)_

Reply:
although it is prettier now i think?

Reply:
I'm fine with the way you ended up doing it.